sdk/src/system_transaction.rs

```diff
@@ -15,10 +15,58 @@ pub fn create_account(
     lamports: u64,
     space: u64,
     program_id: &Pubkey,
+) -> Transaction {
+    generate_create_account_tx(
+        from_keypair,
+        to,
+        recent_blockhash,
+        lamports,
+        space,
+        program_id,
+        false,
+    )
+}
+
+pub fn create_rent_exempted_account(
```

Comment:
is anyone actually using this function?  no need to expose this.  "completeness' sake" isn't a thing for us.

Reply:
This is being used by `wallet`, while creating an executable.